localstack-core/localstack/services/events/target.py

```diff
@@ -181,25 +181,20 @@ def _validate_input(self, target: Target):
             self._validate_input_transformer(input_transformer)
 
     def _initialize_client(self) -> BaseClient:
-        """Initializes internal boto client.
```

Comment:
Why remove the docstring - it makes it easier to understand why we initialize it with the role_arn as source